azure-pipelines.yml

```diff
@@ -300,6 +300,36 @@ jobs:
       msbuildArgs: '/p:AppxPackageSigningEnabled=false'
     displayName: 'Build UWP_x86'
 
+- job: uwp_napi_jsi_x64
+  pool:
+    vmImage: 'windows-latest'
+
+  steps:
+  - script: |
```

Comment:
You don't need to manually invoke the submodule sync. As part of the Azure Pipelines schema you can specify how you want the repo to be cloned:

```yml
steps:
  checkout: self
  clean: true
  submodules: true

  - task: ...
```

Reply:
this should now be fixed

Reply:
fwiw it needs to be:

  - checkout: self
    clean: true
    submodules: recursive